docs/content/stable/yugabyte-platform/troubleshoot/install-upgrade-issues/installer.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Install and upgrade issues on virtual machines
+headerTitle: Install and upgrade issues
+linkTitle: Install and upgrade issues
+description: Troubleshoot issues encountered when installing or upgrading YugabyteDB Anywhere on virtual machines.
+menu:
+  stable_yugabyte-platform:
+    identifier: install-upgrade-installer-issues
+    parent: troubleshoot-yp
+    weight: 10
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../installer/" class="nav-link active">
+      <i class="fa-solid fa-building"></i>
+      YBA Installer</a>
+  </li>
+
+  <li>
+    <a href="../vm/" class="nav-link">
+      <i class="fa-solid fa-cloud"></i>
+      Virtual machine</a>
+  </li>
+
+  <li>
+    <a href="../kubernetes/" class="nav-link">
+      <i class="fa-regular fa-dharmachakra" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
+
+</ul>
+
+Occasionally, you might encounter issues during installation and upgrade of YugabyteDB Anywhere using [YBA Installer](../../../install-yugabyte-platform/install-software/installer/).
+
+If you experience difficulties while troubleshooting, contact [Yugabyte Support](https://support.yugabyte.com).
+
```

Comment:
Do you have any links to add here for more Information similar to the kubernetes page? https://deploy-preview-20714--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/troubleshoot/install-upgrade-issues/kubernetes/